prac_05/state_names.py

```diff
@@ -0,0 +1,49 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
```

Comment:
It's good that you have a doc string here, however I believe this is the old docstring from the template file Lindsay provided us. I recommend removing the old information such as "File needs reformatting" and get rid of the "/CP5632"

Reply:
Thank you I appreciate you pointing that out.